adapters/nft-key/index.js

```diff
@@ -0,0 +1,164 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+
+class NFT_K {
+    // stands for NFT Key
+    constructor() {
+        this.name = "NFTK";
+        this.symbol = null;
+        this.token = "0x0000000000000000000000000000000000000000";
+        this.protocol = "AVAILABLE_PROTOCOLS.AVALANCHE";
+        this.block = 12855192;
+        this.contract = "0x1a7d6ed890b6c284271ad27e7abe8fb5211d0739";
+        this.events = struct Listing {
```

Comment:
The events need to be just event names, not structs from the contract.
Please make sure that the provider passes the tests

Reply:
Feel free to look at our already integrated providers from the main branch as an example

Reply:
https://github.com/dappradar/nft-sales-adapters/issues/89#issue-1354518424